addons/pos_sale/static/src/js/OrderManagementScreen/SaleOrderManagementScreen.js

```diff
@@ -217,6 +217,7 @@ odoo.define('pos_sale.SaleOrderManagementScreen', function (require) {
                         order: this.env.pos.get_order(),
                         product: this.env.pos.db.get_product_by_id(this.env.pos.config.down_payment_product_id[0]),
                         price: down_payment,
+                        tax_ids: this.env.pos.db.get_product_by_id(this.env.pos.config.down_payment_product_id[0]).taxes_id,
```

Comment:
I tested without this and the tax of the product is used, so it does not seems useful

Reply:
I believe a very similar situation is being handled in master rn with coupons (percentage and fixed amounts) I would rather wait to see how it will be handled before trying to have a complete solution here.

the problem is it is not simple how to divide the down payment over multiple products with multiple taxes (sometimes even 1 SOL can have more than 1 tax). also with fixed amount down payment, how much of each SOL will we deduct?

So given that, do you think we shouldn't have any fix for now?

Reply:
What we can probably do as a fix here, is using the tax of the down payment product (most of business have only one tax). But check if this tax is included in the price or not, and depending on that, we'll take the percentage on the total amount or on the untaxed amount.

It will keep the existing behavior, and will also fix your problem when tax is not included (it won't fix when multiple taxes, but we'll do it in master)